external-crates/move/crates/move-analyzer/trace-debug/README.md

```diff
@@ -1,20 +1,42 @@
+Provides the ability to visualize Move trace files, which can be generated for a given package when running Move tests. These trace files contain information about which Move instructions are executed during a given test run. This extension leverages an implementation of the [Debug Adapter Protocol](https://microsoft.github.io/debug-adapter-protocol) (DAP) that analyzes Move execution traces and presents them to the IDE client (in this case a VSCode extension) in a format that the client understands and can visualize using a familiar debugging interface.
+
+## Supported features
+
+Currently we support trace-debugging of Move unit tests only. and the following trace-debugging features are supported:
+- stepping forward through the trace (step, next, step out, and continue commands)
+- tracking local variable values (including enums/structs and references)
+- line breakpoints
+
+Note that support for trace-debugging macros and enums is limited at this point - stepping through macros or code related to enums may result in somewhat unexpected results due to how these constructs are handled internally by the Move execution framework. In particular, variable value tracking may be affected when trace-debugging these constructs. Work is ongoing to improve state-of-the-art - improvement suggestions and bug reports files as issues agains Sui's GitHub [repository](https://github.com/MystenLabs/sui) are greatly appreciated.
 
 # How to Install
 
-When this extension and its companion DAP implementation become more mature, we will make it available in the VSCode Marketplace. At the moment, in order to experiment with it must be built and installed locally (start in the main directory of this extension):
-1. Install dependencies for the extension:
-```bash
-npm install
-```
-2. Install dependencies for the DAP implementation:
-```bash
-npm install --prefix ../move-trace-adapter
-```
-3. Package the extension (it will pull relevant files from the DAP implementation)
-```
-vsce package -o move-trace-debug.vsix
+1. Open a new window in any Visual Studio Code application version 1.61.0 or greater.
+2. Open the command palette (`⇧` + `⌘` + `P` on macOS, or use the menu item *View > Command Palette...*) and
+   type **Extensions: Install Extensions**. This will open a panel named *Extensions* in the
+   sidebar of your Visual Studio Code window.
+3. In the search bar labeled *Search Extensions in Marketplace*, type **Move Trace**. The Move Trace debugger extension
```

Comment:
Can we do "Move Trace or <the fully-qualified name of the extension>"? I would strongly prefer that we have the fully qualified name of the extension just to make it completely obvious.

Reply:
Just to make sure that I got it right - do you want to change **Move Trace** to **Move Trace Debugger** (which is the full version of the extension at this point)?

Reply:
Went with what guessed. Can certainly finessed it further at a later time